packages/url_launcher/url_launcher_android/example/integration_test/url_launcher_test.dart

```diff
@@ -20,4 +20,27 @@ void main() {
     // sms:, tel:, and mailto: links may not be openable on every device, so
     // aren't tested here.
   });
+
+  testWidgets('launch and close', (WidgetTester _) async {
+    final UrlLauncherPlatform launcher = UrlLauncherPlatform.instance;
+
+    // Launch a url then close.
+    expect(
+        await launcher.launch('https://flutter.dev',
```

Comment:
We shouldn't actually launch real external URLs when we can avoid it. See https://github.com/flutter/packages/blob/ac4137624a13c87f20781795b9a56891cdadff11/packages/webview_flutter/webview_flutter/example/integration_test/webview_flutter_test.dart#L25-L43 for a better option for integration tests.

Reply:
Oh thanks I can convert to this.